src/common/ty/ty_printer.ml

```diff
@@ -140,12 +140,6 @@ let type_ ?(size=5000) ?(with_comments=true) t =
 
   and type_alias { ta_name = { provenance; name; _ }; ta_tparams; ta_type } =
     match provenance with
```

Comment:
You don't need to match against provenance here. Just always return `type ...`.

Reply:
okay! should be updated now